codestodians/food-basket-front-end
# Optimize test coverage of all lines of Custom Components

#### What does this PR do?
- It ensures that all lines of the custom components are covered while testing.
- It optimizes the formValidation helper test-suite.

#### Description of Task to be completed?
- Put the unused props of the custom components to use.
- Get rid of redundant props.
- Update utils test suite.
- Add default props for `Text` component.

#### How should this be manually tested?
- Run `yarn test` and notice that all lines of `Box`, `Text` and `Button` components are covered.

#### Any background context you want to provide?
- Failure to use certain props at all in the codebase ensures they aren't tested and affects the overall test coverage. Ensuring that all lines are covered in testing ensures that every unnecessary chunk of code is quickly scrapped and that unforeseen cases are adequately handled.

#### What are the relevant pivotal tracker stories?
[#154021840](https://www.pivotaltracker.com/story/show/154021840)
